src/components/header.js

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'gatsby';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Header = ({ siteTitle }) => (
+    <header
+        style={{
```

Comment:
We should decide on a standardized Styling system. I vote we use CSS.modules  
( https://www.gatsbyjs.org/docs/css-modules/ )

Reply:
Sounds good to me! This style is still the original from the starter so I'll get on changing it over at some point soon